docs/tutorial.md

```diff
@@ -34,10 +34,19 @@
     1. Create a public and private key pair. You'll be prompted to create a
        passphrase. Make sure to save it to the `PHRASE` environment variable.
 
+        Generate new key with `gpg` and extract its fingerprint.
         ```shell
-        gpg --quick-generate-key --yes attestor@example.com
-        gpg --armor --export attestor@example.com > gpg.pub
-        gpg --armor --export-secret-keys attestor@example.com > gpg.priv
+        GPG_OUTPUT="$(gpg --quick-generate-key --yes attestor@example.com)"
```

Comment:
Let's add a similar shell script that does all this to here? Otherwise, it's already not trivial to follow the tutorial.

Reply:
I think simplifying it further is definitely a good idea. I think it will probably worth a separate PR? (we should also consider reusing user's existing `attestor@example.com` key, add step to delete key, etc). Leaving these commands in the tutorial for now probably gives user more transparency when the clean-up scripts, etc are not in place yet?

Reply:
I'm fine with adding this in a separate PR. I'd rather not assume that a user who ran 1 of the tutorials will run the other, and in which order. We can just change the usernames here, if it helps.

Reply:
We agree that we should not assume the order that the tutorials run. With using of fingerprint username is probably not an issue.